### PR TITLE
Merge Feature branch/geo compose into v.next

### PR DESCRIPTION
**Pull request by prupani-7**
## Description
PR to merge feature branch changes to v.next

### CODE REVIEW

**Review by shubham7109 — COMMENTED** [review]
@prupani-7 Looks good, tested by building the sample viewer. 

- Let's make a build version bump here to `arcgisMapsKotlinVersion` and use the latest build tag. 
- One minor comment on show-magnifier.

File: show-magnifier/src/main/java/com/esri/arcgismaps/sample/showmagnifier/components/MapViewModel.kt
[the remark this answers is not in the document]
Comment (prupani-7) [reply show-magnifier/src/main/java/com/esri/arcgismaps/sample/showmagnifier/components/MapViewModel.kt:1]: Wow, wondering how this happened. I thought [this PR change](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/pull/149/files#diff-32f50830bdce46927723464be4fc5c470439820a9f34a51c0f21023d9600d9ac) deleted the file, when we started using composable MapView component.

[the remark this answers is not in the document]
Comment (prupani-7) [reply show-magnifier/src/main/java/com/esri/arcgismaps/sample/showmagnifier/components/MapViewModel.kt:1]: I see it now. Its content were deleted, but the whole file was not marked for deletion.

**Comment by prupani-7** [comment]
thanks @shubham7109 updated the feature-branch/geo-compose branch.

**Comment by shubham7109** [comment]
@prupani-7 Can you address the warning for the show magnifier sample metadata?

**Comment by prupani-7** [comment]
> @prupani-7 Can you address the warning for the show magnifier sample metadata? Seems like there are a couple of others as well.

The checker warnings
( file doesn't exist: ../components/ComposeMapView.kt
file doesn't exist: ../components/MapViewModel.kt ) 
are for the 9 samples that got updated to use composable mapView component, which led to deletion of these files.

I believe these come up in the checker as v.next branch still have these (as part of traditional MapView component) and the feature branch we are trying to merge doesn't. 
While comparing the 2 branch's metadata json file, looks like the checker is trying to locate these files and warn accordingly imo.

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 [versions]
 # ArcGIS Maps SDK for Kotlin version
-arcgisMapsKotlinVersion = "200.4.0-4121"
+arcgisMapsKotlinVersion = "200.4.0-4189"
 # ArcGIS Maps SDK for Kotlin Toolkit version
-arcgisToolkitVersion = "200.4.0-4121"
+arcgisToolkitVersion = "200.4.0-4183"
```
Comment (shubham7109) [inline gradle/libs.versions.toml:5]: Let's use the latest toolkit build here: 
```suggestion
arcgisToolkitVersion = "200.4.0-4189"
```

**Review by shubham7109 — APPROVED** [review]
Looks good, tested using a manual merge and the file changes are okay. Just one minor comment:

**Comment by shubham7109** [comment]
@prupani-7 Remind to use "Squash and merge" when closing this PR! ⚠️

**Comment by prupani-7** [comment]
Thanks for review! @shubham7109 
fyi, it auto deleted the feature-branch/geo-compose

**Comment by prupani-7** [comment]
> Reminder to use "Squash and merge" when closing this PR! ⚠️

I use squash and merge always when merging.
But any particular reason you are mentioning it for?

**Comment by shubham7109** [comment]
No particular season, just a heads up since this PR brings in about 90 commits. Thanks @prupani-7

**Pull request merged**